vtex/faststore
# Feat: Adds `Toast` Component

## What's the purpose of this pull request?

- [x] Moves `Toast` component to @faststore/components 
- [x] Adds `Toast` styles on @faststore/ui
- [x] Review `Toast` structure
- [x] Updates @faststore/core with new `Toast`
- [x] adds documentation in apps/site package

It also fixes the Toast position in mobile devices while using the docs:
|Before|After|
|-|-|
|<img src="https://user-images.githubusercontent.com/11325562/226780961-6e2c4a95-cd78-4542-ac28-98d8ac3b95f3.png" width="600"/>|![NP5dakfiTN](https://user-images.githubusercontent.com/11325562/226780621-1ab859e5-cade-4436-a23c-a6444ad68a9b.gif)|



## How to test it?
- You can see the components page in our nextra doc from the [preview](https://faststore-site-pih42fgxf-faststore.vercel.app/components/molecules/toast).
- In the store you can run `yarn dev` and checks the `Toast` component locally or test it using the [preview link](https://sfj-279515c--starter.preview.vtex.app/).
You can use the subscription feature in home or even mock messages inside the common `Toast` component.

### Starter PR
https://github.com/vtex-sites/starter.store/pull/22
